priv/repo/migrations/20200420133108_create_tags.exs

```diff
@@ -0,0 +1,15 @@
+defmodule AppApi.Repo.Migrations.CreateTags do
+  use Ecto.Migration
+
+  def change do
+    create table(:tags) do
+      add :text, :string
+      add :id_person, :integer
+
+      timestamps()
+    end
+
+    create unique_index(:tags, [:text])
```

Comment:
This index makes sure the database doesn't contain tags with the same name

Reply:
@SimonLab which issue requirement or acceptance criteria is this for?

Reply:
I've tried to follow the requirements from https://github.com/dwyl/app/issues/245

Reply:
@SimonLab thanks for clarifying. In which case it could be worth having that issue linked in the PR description and having it "in-progress" on the Kanban board (_which makes it easier for the reviewer_) 😉